README.md

```diff
@@ -301,6 +301,21 @@ visible via dynamic_reconfigure, see section [Configuration](#configuration). Wh
 service has returned you can invoke the [capture_2d](#capture) service to trigger a 2D capture
 using these settings.
 
+### capture_and_save_frame
+[zivid_camera/CaptureAndSaveFrame.srv](./zivid_camera/srv/CaptureAndSaveFrame.srv)
+
+Invoke this service to trigger a 3D capture and save the frame as ZDF file.
```

Comment:
Consider to just refer back to the capture service section here?
Something ala "Does the same as the capture service, additionally it saves the frame to the [...]"
Then there is 1 less place to update when we add new topics etc.